src/styles/Resizer.scss

```diff
@@ -2,41 +2,41 @@
 
 .Resizer {
   background: #000;
+  background-clip: padding;
+  background-clip: padding;
+  background-clip: padding-box;
+  box-sizing: border-box;
+  box-sizing: border-box;
+  box-sizing: border-box;
```

Comment:
```suggestion
  background-clip: padding-box;
  box-sizing: border-box;
```

I'm actually a little worried about this change since `background-clip` actually [takes different values](https://developer.mozilla.org/en-US/docs/Web/CSS/background-clip) than the per-browser webkit properties.

Reply:
hm ill disable the rule then

Reply:
i disabled the rule that caused this and reverted this to what it was originally!